package.json

```diff
@@ -17,10 +17,10 @@
     "build": "tsc",
     "lint": "npm-run-all lint:es lint:ts",
     "lint:ts": "tsc --isolatedModules --noEmit",
-    "start": "npm run build && npx probot run ./dist/bin/main.js",
+    "start": "npm run build && node ./dist/bin/main.js",
```

Comment:
Please remove unrelated changes.

Reply:
I am using node v14 to run this project, but this start script fails.
I have to change some lines, now, it just works. I have token a look at probot, so, what is the point here?
Does it make sense to make the project more robust with less dependencies, what is the particular reason to use probot here? It breaks the start command.
@huan

Reply:
The changes are related, because I have to start the project afterward. They have to be in the main stream, by the way, I am using Windows. So, now the scripts in package.json are compatible with Windows.